tests/static/form/missing-value.xml

```diff
@@ -0,0 +1,32 @@
+<?xml version="1.0" encoding="UTF-8" ?>
+<h:html xmlns:h="http://www.w3.org/1999/xhtml" xmlns:orx="http://openrosa.org/jr/xforms" xmlns="http://www.w3.org/2002/xforms" xmlns:xsd="http://www.w3.org/2001/XMLSchema" xmlns:jr="http://openrosa.org/javarosa" xmlns:vellum="http://commcarehq.org/xforms/vellum">
+	<h:head>
+		<h:title>Untitled Form</h:title>
+		<model>
+			<instance>
+				<data xmlns:jrm="http://dev.commcarehq.org/jr/xforms" xmlns="http://openrosa.org/formdesigner/510328F4-59B2-4168-ABF7-F96FA40BB790" uiVersion="1" version="1" name="Untitled Form">
+					<choice />
+				</data>
+			</instance>
+			<bind vellum:nodeset="#form/choice" nodeset="/data/choice" />
+			<itext>
+				<translation lang="en" default="">
+					<text id="choice-label">
+						<value>choice</value>
+					</text>
+					<text id="choice-empty-label">
+						<value>empty</value>
+					</text>
+				</translation>
+			</itext>
+		</model>
+	</h:head>
+	<h:body>
+		<select1 vellum:ref="#form/choice" ref="/data/choice">
+			<label ref="jr:itext('choice-label')" />
+			<item>
+				<label ref="jr:itext('choice-empty-label')" />
```

Comment:
Just so I'm clear, the issue with this form is that this label doesn't have a sibling value node, right?

Reply:
correct